etc/build/install-minizinc.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# version as of 16 November 2023
```

Comment:
looks good! I would just put today's date here in the comment. and checkout 2.8.3 which is the latest tag on github. we don't want to track latest on their main branch - would be bad for reproducibility. some other install-*.sh scripts do this.

Reply:
Makes sense! I have updated the script to do so